addons/weather/functions/fnc_getMapData.sqf

```diff
@@ -47,9 +47,22 @@ GVAR(currentTemperature) = 15;
 GVAR(currentHumidity) = 0;
 GVAR(currentOvercast) = 0;
 
-// Get all non inherited arrays to filter maps that inherit from Stratis/Altis/Tanoa
 private _cfgPath = configFile >> "CfgWorlds" >> _worldName;
+
+// Check if values are set in mission since CfgWorlds is valid in description.ext
+private _missionCfgPath = missionConfigFile >> "CfgWorlds" >> _worldName;
```

Comment:
Imo the specific world should take precedence over `Any`.

Reply:
Yeah that's fair, I'll swap them

Reply:
Done